### PR TITLE
Fix angular-animate definitions

### PR DESCRIPTION
**Pull request by ghost**
https://code.angularjs.org/1.5.6/docs/api/ngAnimate/service/$animateCss
- Rename transition to transitionStyle and keyframe to keyFrameStyle
- Add missing structural and cleanupStyles properties

### CODE REVIEW

File: angularjs/angular-animate.d.ts [hunk 1]
```diff
@@ -250,11 +256,16 @@ declare namespace angular.animate {
         /**
          * The numeric index representing the stagger item (e.g. a value of 5 is equal to the sixth item
          * in the stagger; therefore when a stagger option value of 0.1 is used then there will be a stagger delay of 600ms)
-         * applyClassesEarly - Whether or not the classes being added or removed will be used when detecting the animation.
```
Comment (adidahiya) [inline angularjs/angular-animate.d.ts]: looks like `applyClassesEarly` is a valid field in this interface -- would you mind adding it back as a proper field (not a comment)?

**Comment by adidahiya** [comment]
overall lgtm 👍 

would be nice to address my small comment, but it doesn't need to block a merge

**Comment by mhegazy** [comment]
@stijnvn can you please get another PR with the proposed fixed?

**Comment by mhegazy** [comment]
thanks @adidahiya for reviewing!

**Pull request merged**